components/Funnels/TikTokAccountAnalysis/ChainOfThought.tsx

```diff
@@ -0,0 +1,58 @@
+import ChatInput from "@/components/Chat/ChatInput";
+import Messages from "@/components/Chat/Messages";
+import { useTikTokAnalysisProvider } from "@/providers/TIkTokAnalysisProvider";
+import { STEP_OF_ANALYSIS } from "@/types/Thought";
+import { ScrollTo } from "react-scroll-to";
+import Segments from "./Segments";
+
+const ChainOfThought = () => {
+  const { thought, username, progress, result, segments } =
+    useTikTokAnalysisProvider();
+
+  return (
+    <main className="flex-1 flex">
+      <div className="max-w-3xl mx-auto w-full h-screen mx-auto pt-10 flex flex-col">
+        <div
+          className={`flex gap-2 ${thought === STEP_OF_ANALYSIS.FINISHED ? "items-start" : "items-center"}`}
+        >
+          {/* eslint-disable-next-line @next/next/no-img-element */}
+          <img
+            src={"/logo-light.png"}
+            alt="not found logo"
+            className="!w-6 !h-6"
+          />
+          <p className="text-sm">
+            {thought === STEP_OF_ANALYSIS.FINISHED
+              ? result?.videos?.length
+                ? `Analysis complete! @${username.replaceAll("@", "")}’s is ${result?.nickname} and makes content in ${result?.region}. He has ${result?.fans} followers. \nPlease select a fan segmentation below to generate a report for brand partnership deals.`
```

Comment:
Do not assume gender.
```suggestion
                ? `Analysis complete! @${username.replaceAll("@", "")}’s is ${result?.nickname} and makes content in ${result?.region}. They have ${result?.fans} followers. \nPlease select a fan segmentation below to generate a report for brand partnership deals.`
```